core/includes/bootstrap.inc

```diff
@@ -2290,17 +2290,8 @@ function watchdog_severity_enabled($severity = WATCHDOG_NOTICE) {
  *   NULL if message is already translated or not possible to
  *   translate.
  * @param $severity
- *   The severity of the message; one of the following values as defined in
- *   @link http://www.faqs.org/rfcs/rfc3164.html RFC 3164: @endlink
- *   - WATCHDOG_EMERGENCY: Emergency, system is unusable.
- *   - WATCHDOG_ALERT: Alert, action must be taken immediately.
- *   - WATCHDOG_CRITICAL: Critical conditions.
- *   - WATCHDOG_ERROR: Error conditions.
- *   - WATCHDOG_WARNING: Warning conditions.
- *   - WATCHDOG_NOTICE: (default) Normal but significant conditions.
- *   - WATCHDOG_INFO: Informational messages.
- *   - WATCHDOG_DEBUG: Debug-level messages.
- *   - WATCHDOG_DEPRECATED: Use of deprecated functions or features.
+ *   The severity of the message; see watchdog_severity_levels() for
+ *   possible values.
```

Comment:
I think this docblock is worth duplicating the list of levels, since `watchdog()` is the primary function called where the value needs to be specified. So let's revert this change while leaving the current `@see watchdog_severity_levels()` in place.

```suggestion
 *   The severity of the message; one of the following values as defined in
 *   @link http://www.faqs.org/rfcs/rfc3164.html RFC 3164: @endlink
 *   - WATCHDOG_EMERGENCY: Emergency, system is unusable.
 *   - WATCHDOG_ALERT: Alert, action must be taken immediately.
 *   - WATCHDOG_CRITICAL: Critical conditions.
 *   - WATCHDOG_ERROR: Error conditions.
 *   - WATCHDOG_WARNING: Warning conditions.
 *   - WATCHDOG_NOTICE: (default) Normal but significant conditions.
 *   - WATCHDOG_INFO: Informational messages.
 *   - WATCHDOG_DEBUG: Debug-level messages.
 *   - WATCHDOG_DEPRECATED: Use of deprecated functions or features.
```